assets/src/web-stories-block/block/block.json

```diff
@@ -1,6 +1,18 @@
 {
+  "apiVersion": 2,
   "name": "web-stories/embed",
+  "title": "Web Stories",
+  "description": "Embed Web Stories.",
```

Comment:
Hm, this was translatable before, how would it be handled now?

Reply:
WordPress.org would handle this, but it‘s not implemented yet.

Reply:
> WordPress.org would handle this, but it‘s not implemented yet.

How does that work?

Reply:
translate.wordpress.org uses WP-CLI to extract strings from plugins and themes, including `block.json` and soon `theme.json`. So all translatable fields in `block.json` will be part of the POT file.

`register_block_type_from_metadata()` then translates the fields on the fly, picking these translations from the POT file.

Unfortunately translate.wordpress.org did not actually do this until 2 days ago, see https://meta.trac.wordpress.org/ticket/5737.

Reply:
That being said, this only works on the server-side, not on the client. For client side, `registerBlockTypeFromMetadata` would need to be used.

See https://github.com/WordPress/gutenberg/blob/722cd5c3b7ae888ac53a6ce6c69b3a0a9e4e9276/docs/reference-guides/block-api/block-metadata.md

Reply:
Wow, that is super disappointing that wasn't work. I have reverted the change and added some field to registration.